sotodlib/mapmaking/tod_grouping.py

```diff
@@ -0,0 +1,200 @@
+"""
+This class intends to produce a obslists dictionary that will be feed to the mapmaker. Every element of obslists will be mapped
+as an individual map. For example, we could request depth-1 maps which are continous scans one after the other that will be mapped
+together as a single maps, we could request per-obs which will be individual observations mapped individually, etc.
+"""
+import numpy as np
+from pixell import utils
+from scipy import ndimage
+
+from .utilities import get_ids
+
+def find_scan_periods(obs_info, ttol=60, atol=2*utils.degree, mindur=120):
```

Comment:
I do not think this function is necessary.
How about just use the obsdb? I think all query stuff is doable with obsdb with more flexible way.

Reply:
Even if their is any thing we can not do with obsdb, we should improve the obsdb first.